SU2_CFD/src/solvers/CSpeciesFlameletSolver.cpp

```diff
@@ -174,41 +123,39 @@ void CSpeciesFlameletSolver::SetInitialCondition(CGeometry** geometry, CSolver**
 
     const su2double flamenorm = GeometryToolbox::Norm(nDim, flame_normal);
     const su2double temp_inlet = config->GetInc_Temperature_Init();
-    su2double prog_inlet = config->GetSpecies_Init()[I_PROGVAR];
     su2double enth_inlet = config->GetSpecies_Init()[I_ENTH];
 
-    su2double prog_burnt;
-    su2double prog_unburnt = 0.0;
+    su2double prog_burnt = 0, prog_unburnt, point_loc;
+    su2double *scalar_init = new su2double[nVar];
```

Comment:
Is there a MAXNVAR you can use to make a static array instead of using new / delete?